commands/cluster-slot-stats.md

```diff
@@ -0,0 +1,90 @@
+Returns an array of slot usage statistics for slots assigned to the current shard.
+The command is suitable for Valkey Cluster users aiming to assess general slot usage trends, identify hot / cold slots, migrate slots for a balanced cluster workload, and / or re-write application logic to better utilize slots.
+
+The following statistics are supported:
+
+* `key-count` -- Number of keys residing in a given slot.
+* `cpu-usec` -- Amount of CPU time (in microseconds) spent on a given slot.
+* `network-bytes-in` -- Amount of network ingress (in bytes) received for given slot.
+* `network-bytes-out` -- Amount of network egress (in bytes) sent out for given slot.
+
+## Options
+
+* `ORDERBY` -- Returns an ordered slot statistics based on the specified statistic and sub-arguments to identify hot / cold slots across the cluster. In the event of a tie in the stats, ascending slot number is used as a tie breaker.
```

Comment:
In the past we talked about also including the descriptions for limit and ASC/DESC. Specifically to implement pagination if someone wanted to do that.